documentation/book/proc-configuring-kafka-mirror-maker-whitelist.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-mirror-maker-whitelist.adoc
+
+[id='proc-configuring-kafka-mirror-maker-whitelist-{context}']
+= Configuring the topics whitelist
+
+The topic whitelist, describing which topics have to be mirrored by Kafka Mirror Maker from source to target Kafka cluster, can be configured using the `whitelist` property in `KafkaMirrorMaker.spec`.
```

Comment:
Rewrite to:

Specify the list topics that have to be mirrored by the Kafka Mirror Maker from source to target Kafka cluster using the `whitelist` property in `KafkaMirrorMaker.spec`.